static/js/lib/ckeditor/plugins/util.ts

```diff
@@ -48,7 +48,7 @@ const ensureEncasedInQuotes = (text: string, singleQuotes = false) => {
   return q + text + q
 }
 
-export function buildAttrsString(attrs: RegExpMatchArray | null): string {
+export function buildAttrsString(attrs: string[] | null): string {
```

Comment:
Typescript 4.9 introduced a small breaking change: Previously `string[]` was assignable to `RegExpMatchArray`; it no longer is.
- We call `buildAttrsString` with both `string[]` and `RegExpMatchArray` argument types
- `string[]` is the looser type (`RegExpMatchArray` satisfies the `string[]` interface (and has some extra properties like `.groups`, too). We only use the `string[]`) and is sufficient for our purposes, so let's just use that.

See also https://github.com/microsoft/TypeScript/pull/49682#discussion_r915601446